docs/painless/painless-contexts/painless-weight-context.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[painless-weight-context]]
+=== Weight Context
+
+Use a Painless script to create a
+{es_version}/index-modules-similarity.html[weight] for use in a
+<<painless-similarity-context, similarity script>>.  Weight is used to prevent
+re-calculation of constants that remain the same across documents.
+
```

Comment:
Again, just close it up, it's not necessary to hyphenate.

Reply:
Fixed.